packages/circuit-ui/components/Calendar/components/CalendarWrapper/CalendarWrapper.spec.js

```diff
@@ -16,20 +16,8 @@
 import CalendarWrapper from '.';
 
 describe('CalendarWrapper', () => {
-  /**
-   * Style tests.
-   */
   it('should render with default styles', () => {
     const actual = create(<CalendarWrapper />);
     expect(actual).toMatchSnapshot();
   });
-
-  /**
-   * Accessibility tests.
-   */
-  it('should meet accessibility guidelines', async () => {
```

Comment:
Just wondering, is it ok to remove this test?

Reply:
Yes, because this component only adds global styles. This test didn't make much sense to begin with.